setup.py

```diff
@@ -28,7 +28,7 @@
     long_description_content_type="text/markdown",
     packages=find_packages(exclude=["tests*", "sandbox"]),
     install_requires=[
-        "pylint<3",
```

Comment:
Is that a required change? Let's keep it to `<4`-only?

Reply:
Let's make it `>=2` together with some (local, for now) "todo: maybe-remove?" on lines like e.g.

https://github.com/pylint-dev/pylint-pytest/blob/fc8fa90c2a0b67e9cca52efaf9253fd2f0690a7c/pylint_pytest/checkers/fixture.py#L317

**but on v2**. If you'd like this PT on v1, let's please keep it "all the way compatible".

---

On v2, we can be more aggressive for sure.

FWIW, **I don't expect/want you** to tox-matrix-check pylint v1.
I just want to best-effort support whatever people might have been doing 🙏

Reply:
Ok, it does look like we can fo for `>=2`, specially looking at https://github.com/pylint-dev/pylint-pytest/actions/runs/7059395001/job/19216868149?pr=27, seems like the current master does not support v1 anymore

Reply:
Given your investigation, would you consider blowing up

```py
if int(pylint.__version__.split(".")[0]) < 2:
```

and

```py
if PYLINT_VERSION_MAJOR < 2
```

too?

Please the change it in your RP-subject

Reply:
Resolved in 1195f5c68ee4f6631fdfd1e0112309d691b8ec62

Reply:
_I think I also found a completed action; definitely proving pylint v1 is not supported: https://github.com/pylint-dev/pylint-pytest/actions/runs/7078716431/job/19264514785_